e2e-tests/cypress/tests/support/ui/playbooks.js

```diff
@@ -176,7 +176,7 @@ Cypress.Commands.add('updateStatus', (message, reminderQuery) => {
 
         if (reminderQuery) {
             cy.get('#reminder_timer_datetime').within(() => {
-                cy.get('input').type(reminderQuery, {delay: TIMEOUTS.TWO_HUNDRED_MILLIS, force: true}).type('{enter}', {force: true});
+                cy.get('input').type(reminderQuery, {delay: TIMEOUTS.TWO_HUNDRED_MILLIS, force: true}).wait(TIMEOUTS.ONE_SEC).type('{enter}');
```

Comment:
Adding a wait here allows the react-select component to suggest something and in turn let us "select" it. It used to pass before, but we had to upgrade react-select with the mono-repo and I'm wondering if it regressed somehow. Icky, but working now.